README.md

```diff
@@ -17,12 +17,31 @@ To install the dependencies we provide a Dockerfile:
 ```bash
 docker build -t chat-with-nerf:latest .
 ```
+Or if you want to pull remote image from Dockerhub to save significant time, please try:
+```bash
+docker pull jedyang97/chat-with-nerf:latest
+```
+
+Otherwise, if you prefer build it locally:
+```bash
+conda create --name nerfstudio -y python=3.8
+conda activate nerfstudio
+pip install torch==1.13.1 torchvision functorch --extra-index-url https://download.pytorch.org/whl/cu117
+pip install ninja git+https://github.com/NVlabs/tiny-cuda-nn/#subdirectory=bindings/torch
+pip install nerfstudio
+
+git clone https://github.com/kerrj/lerf
+python -m pip install -e .
+ns-train -h
+```
+Note that specific CUDA 11.3 is required. For further information, please check NeRFStudio installation
+guide.
 
 Then locally you need to run
 ```bash
 git clone https://github.com/sled-group/chat-with-nerf.git
 ```
-Run the Docker using the following fashion, especially mount chat-with-nerf under workspace
+If using Docker, you can utilize the following fashion, especially mount chat-with-nerf under workspace
```

Comment:
"If using Docker, you can use the following command to spin up a docker container with `chat-with-nerf` mounted under `/workspace`:"